Tests/PackageRegistryTests/RegistryClientTests.swift

```diff
@@ -1615,31 +1517,15 @@ final class RegistryClientTests: XCTestCase {
             let manifest = try await registryClient.getManifestContent(
                 package: identity,
                 version: version,
-                customToolsVersion: nil
+                customToolsVersion: toolsVersion
             )
             let parsedToolsVersion = try ToolsVersionParser.parse(utf8String: manifest)
-            XCTAssertEqual(parsedToolsVersion, .current)
-        }
-
-        do {
-            let manifest = try await registryClient.getManifestContent(
-                package: identity,
-                version: version,
-                customToolsVersion: .v5_3
-            )
-            let parsedToolsVersion = try ToolsVersionParser.parse(utf8String: manifest)
-            XCTAssertEqual(parsedToolsVersion, .v5_3)
+            #expect(parsedToolsVersion == toolsVersion ?? .current)
```

Comment:
suggestion (non-blocking): In order to be more explicit, the value of the expected passed tools version can be passed in as the test data, instead of using `toolsVersion ?? .current)

Reply:
This exercises that "nil" is interpreted as .current, so was deliberate

Reply:
I understand the test intent, but the test can look cleaner as a parameterized test.

e.g. 
```
@Test(
    arguments: [
        (toolsVersion: ToolsVersion.v5_3, expectedParsedVersion: ToolsVersion.v5_3),
        (toolsVersion: nil, expectedParsedVersion: ToolsVersion.current),
    ]
)
func mytest(toolsVersion: ToolsVersion?, expectedParsedVersion: ToolsVersion) async throws {
    ...
    #expect(parsedToolsVersion == expectedParsedVersion)
}
```